release/src/main/scripts/build_release_candidate.sh

```diff
@@ -288,14 +288,15 @@ if [[ $confirmation = "y" ]]; then
   mkdir -p ${LOCAL_WEBSITE_REPO}
 
   echo "------------------Building Python Doc------------------------"
-  virtualenv ${LOCAL_PYTHON_VIRTUALENV}
-  source ${LOCAL_PYTHON_VIRTUALENV}/bin/activate
+  python3 -m venv "${LOCAL_PYTHON_VIRTUALENV}"
```

Comment:
Is this approach generally preferred over `virtualenv --python=python3` (just asking for my own edification)?

Reply:
The new apprach works starting from Py3 as venv became released as part of the standard library and is now documented in Python docs: https://docs.python.org/3/library/venv.html. Old approach also works if virtualenv is installed.